src/app/public/modules/ag-grid/cell-editors/cell-editor-datepicker/cell-editor-datepicker.component.html

```diff
@@ -5,7 +5,6 @@
 >
   <sky-datepicker
```

Comment:
The datepicker button can only be opened with `space`, but not the `enter` key. Is that expected?

Reply:
kind of? Hitting `enter` is the only way to either start editing a cell or save the edits made via the keyboard. I could try to suppress the event from the datepicker editor when the button is focused if we think it is important to be able to use `enter` or `space`. My initial reason for not doing so was just trying to avoid complicating keyboard interactions even more.

Reply:
I'm not sure of the right answer here. We might have to leave this one up to an accessibility audit. I'm going to resolve this thread as its not a dealbreaker, IMO.